### PR TITLE
Show selected date ranges on subtitles

### PR DESCRIPTION
**Pull request by spacephoton**
CleanShot 2022-03-20 at 21 20 18
CleanShot 2022-03-20 at 21 22 03

### CODE REVIEW

**Comment by spacephoton** [comment]
do you know what the Jest issue could be ?

**Comment by chaoran-chen** [comment]
I think that `react-bootstrap/esm/Form` is wrong. It should be `react-bootstrap/Form`.

File: src/widgets/VariantInternationalComparisonChart.tsx [hunk 1]
```diff
@@ -109,6 +111,8 @@ export const VariantInternationalComparisonChart = ({
       isFixed: false,
     }))
   );
+  const exploreUrl = useExploreUrl();
```
Comment (chaoran-chen) [inline src/widgets/VariantInternationalComparisonChart.tsx]: The `...Chart` components should be (more or less) pure and are not supposed to fetch information through `useExploreUrl()`. Instead, they should only use information that is passed through the props. This is causing the current errors.

**Comment by spacephoton** [comment]
Ah I see, so only this chart component right? was little confused so thanks!

**Comment by anastasia-escher** [comment]
What's about this PR?
https://github.com/cevo-public/cov-spectrum-website/pull/435
It's been there for so long.

**Pull request closed without merging**